README.md

```diff
@@ -7,7 +7,7 @@ Auditing for TLS certificates.
 
 ## Dependencies ##
 
- - [OpenSSL](https://www.openssl.org/source/), at least 1.0.0, preferably 1.0.2 (and up)
+ - [OpenSSL](https://www.openssl.org/source/), at least 1.0.0q, preferably 1.0.1l or 1.0.2 (and up)
```

Comment:
Correct, but I've tested with 1.0.1l and there's no reason to point to an even older release.